src/_includes/docs/install/reqs/linux/software.md

```diff
@@ -46,6 +46,13 @@ To develop Flutter on {{include.os}}:
           libbz2-1.0:i386
       ```
 
+      > If your Ubuntu distribution can't find these packages, you might need to enable multi-architecture support
```

Comment:
```suggestion
If your Ubuntu distribution can't find the following packages,
you might need to enable multi-architecture support:
```

Reply:
Good suggestion. Because the packages have already been listed, would you suggest moving this paragraph above the `console` code block or possibly saying "...can't find the listed packages"?